tests/unit/type/hash.tcl

```diff
@@ -648,6 +648,31 @@ start_server {tags {"hash"}} {
         }
     }
 
+    test {HINCRBYFLOAT over hash-max-listpack-value encoded with a listpack} {
+        set original_max_value [lindex [r config get hash-max-ziplist-value] 1]
+        r config set hash-max-ziplist-value 8
```

Comment:
the title mentions `listpack` and the code uses `ziplist`.
these are aliases to the same config (ziplist is the old term and listpack is the new).
let's use listpack in new "code" we add